### ISSUE TITLE
Document how to get started contributing in Eclipse

### ISSUE THREAD

**Issue by pilotniq** [knowm/XChange#458]
I'd like some documentation on how to set up the XChange project in Eclipse.

I've checked out the develop branch in git. I'm not sure if I should set up one project for the whole thing, or one project for each subdirectory (I tried the latter, but got problems with a Maven reference to xchange-parent).

I have unresolved references to si.mazi.rescu.ParamsDigest for example. Shouldn't the POM fix these references?

I also had problems with references to javax.ws... packages (like javax.ws.rs.Consumes). I'm not sure if I should configure a Project Facet to fix this.

Basically, a web page in the Wiki detailing how to set up Eclipse properly would save time for potential contributors and make it easier to contribute!
